modules/cnf-fast-event-notifications-api-reference.adoc

```diff
@@ -63,6 +69,51 @@ Creates a new subscription. If a subscription is successfully created, or if it
 }
 ----
 
+.Example PTP lock-state events subscription payload
+[source,json]
+----
+{
+"endpointUri": "http://localhost:8989/event",
+"resource": "/cluster/node/{node_name}/sync/ptp-status/lock-state"
+}
+----
+
+.Example PTP os-clock-sync-state events subscription payload
+[source,json]
+----
+{
+"endpointUri": "http://localhost:8989/event",
+"resource": "/cluster/node/{node_name}/sync/sync-status/os-clock-sync-state"
+}
+----
+
+.Example PTP ptp-clock-class-change events subscription payload
+[source,json]
+----
+{
+"endpointUri": "http://localhost:8989/event",
+"resource": "/cluster/node/{node_name}/sync/ptp-status/ptp-clock-class-change"
```

Comment:
should be `/sync/ptp-status/clock-class`.

Do not mix `type` and `source`. Since you are referring all other events by source, might as well use source name for this event too. This applies to most other places that `ptp-clock-class-change` are referenced.

event type: `event.sync.ptp-status.ptp-clock-class-change`
event source: `/sync/ptp-status/clock-class`